examples/docs_snippets/docs_snippets/integrations/dbt/dbt.py

```diff
@@ -42,19 +42,32 @@ def scope_dbt_cli_resource_config():
 
 
 def scope_schedule_assets():
-    dbt_assets = []
+    from dagster import file_relative_path, with_resources
+    from dagster_dbt import dbt_cli_resource, load_assets_from_dbt_project
+
+    DBT_PROJECT_PATH = file_relative_path(
+        __file__, "../../../../assets_dbt_python/dbt_project"
+    )
+    dbt_assets = with_resources(
```

Comment:
move resources to resources arg of Definitions?

Reply:
this isn't user-facing. moved to test